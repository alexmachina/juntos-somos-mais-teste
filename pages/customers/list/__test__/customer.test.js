import React from "react";
import Customers from "../";
import renderer from "react-test-renderer";

describe("Components", () => {
  describe("Customers", () => {
    it("Renders correctly", () => {
      const tree = renderer.create(<Customers />).toJSON();
      expect(tree).toMatchSnapshot();
    });
  });
});
