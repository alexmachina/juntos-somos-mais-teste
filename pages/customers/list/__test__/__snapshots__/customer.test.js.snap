// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Customers Renders correctly 1`] = `
<div
  className="MuiGrid-root makeStyles-container-2 MuiGrid-container"
>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
  >
    <h2
      className="MuiTypography-root MuiTypography-h2 MuiTypography-gutterBottom"
    >
      Usuários Por Região
    </h2>
  </div>
  <div
    className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
  >
    <div
      className="MuiGrid-root makeStyles-container-137 MuiGrid-container MuiGrid-justify-xs-center"
    >
      <div
        className="MuiGrid-root makeStyles-stepperContainer-138 MuiGrid-item MuiGrid-grid-xs-12"
      >
        <div
          className="MuiPaper-root MuiPaper-elevation0 MuiMobileStepper-root MuiMobileStepper-positionStatic"
        >
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall Mui-disabled Mui-disabled"
            disabled={true}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={-1}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M15.41 16.59L10.83 12l4.58-4.59L14 6l-6 6 6 6 1.41-1.41z"
                />
              </svg>
              Back
            </span>
          </button>
          <div
            className="MuiMobileStepper-dots"
          />
          <button
            className="MuiButtonBase-root MuiButton-root MuiButton-text MuiButton-sizeSmall"
            disabled={false}
            onBlur={[Function]}
            onClick={[Function]}
            onDragLeave={[Function]}
            onFocus={[Function]}
            onKeyDown={[Function]}
            onKeyUp={[Function]}
            onMouseDown={[Function]}
            onMouseLeave={[Function]}
            onMouseUp={[Function]}
            onTouchEnd={[Function]}
            onTouchMove={[Function]}
            onTouchStart={[Function]}
            tabIndex={0}
            type="button"
          >
            <span
              className="MuiButton-label"
            >
              Next
              <svg
                aria-hidden="true"
                className="MuiSvgIcon-root"
                focusable="false"
                role="presentation"
                viewBox="0 0 24 24"
              >
                <path
                  d="M8.59 16.59L13.17 12 8.59 7.41 10 6l6 6-6 6-1.41-1.41z"
                />
              </svg>
            </span>
          </button>
        </div>
      </div>
    </div>
  </div>
  <div
    className="MuiGrid-root makeStyles-content-3 MuiGrid-item MuiGrid-grid-xs-12"
  >
    <div
      className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-4"
    >
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-3"
      >
        <div
          className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-2"
        >
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <div
              className="MuiPaper-root MuiPaper-elevation1 makeStyles-filters-203 MuiPaper-rounded"
            >
              <div>
                <header>
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                  >
                    Usuário
                  </h6>
                </header>
                <article>
                  <div
                    className="MuiFormGroup-root"
                  >
                    <label
                      className="MuiFormControlLabel-root"
                    >
                      <span
                        aria-disabled={false}
                        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-218 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                        onBlur={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={null}
                      >
                        <span
                          className="MuiIconButton-label"
                        >
                          <input
                            className="PrivateSwitchBase-input-221"
                            data-indeterminate={false}
                            onChange={[Function]}
                            type="checkbox"
                            value="special"
                          />
                          <svg
                            aria-hidden="true"
                            className="MuiSvgIcon-root"
                            focusable="false"
                            role="presentation"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                      >
                        Especial
                      </span>
                    </label>
                    <label
                      className="MuiFormControlLabel-root"
                    >
                      <span
                        aria-disabled={false}
                        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-218 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                        onBlur={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={null}
                      >
                        <span
                          className="MuiIconButton-label"
                        >
                          <input
                            className="PrivateSwitchBase-input-221"
                            data-indeterminate={false}
                            onChange={[Function]}
                            type="checkbox"
                            value="normal"
                          />
                          <svg
                            aria-hidden="true"
                            className="MuiSvgIcon-root"
                            focusable="false"
                            role="presentation"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                      >
                        Normal
                      </span>
                    </label>
                  </div>
                </article>
              </div>
            </div>
          </div>
          <div
            className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12"
          >
            <div
              className="MuiPaper-root MuiPaper-elevation1 makeStyles-filters-231 MuiPaper-rounded"
            >
              <div>
                <header>
                  <h6
                    className="MuiTypography-root MuiTypography-h6"
                  >
                    Região
                  </h6>
                </header>
                <article>
                  <div
                    className="MuiFormGroup-root"
                  >
                    <label
                      className="MuiFormControlLabel-root"
                    >
                      <span
                        aria-disabled={false}
                        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-218 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                        onBlur={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={null}
                      >
                        <span
                          className="MuiIconButton-label"
                        >
                          <input
                            className="PrivateSwitchBase-input-221"
                            data-indeterminate={false}
                            onChange={[Function]}
                            type="checkbox"
                            value="norte"
                          />
                          <svg
                            aria-hidden="true"
                            className="MuiSvgIcon-root"
                            focusable="false"
                            role="presentation"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                      >
                        Norte
                      </span>
                    </label>
                    <label
                      className="MuiFormControlLabel-root"
                    >
                      <span
                        aria-disabled={false}
                        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-218 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                        onBlur={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={null}
                      >
                        <span
                          className="MuiIconButton-label"
                        >
                          <input
                            className="PrivateSwitchBase-input-221"
                            data-indeterminate={false}
                            onChange={[Function]}
                            type="checkbox"
                            value="nordeste"
                          />
                          <svg
                            aria-hidden="true"
                            className="MuiSvgIcon-root"
                            focusable="false"
                            role="presentation"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                      >
                        Nordeste
                      </span>
                    </label>
                    <label
                      className="MuiFormControlLabel-root"
                    >
                      <span
                        aria-disabled={false}
                        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-218 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                        onBlur={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={null}
                      >
                        <span
                          className="MuiIconButton-label"
                        >
                          <input
                            className="PrivateSwitchBase-input-221"
                            data-indeterminate={false}
                            onChange={[Function]}
                            type="checkbox"
                            value="centro-oeste"
                          />
                          <svg
                            aria-hidden="true"
                            className="MuiSvgIcon-root"
                            focusable="false"
                            role="presentation"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                      >
                        Centro Oeste
                      </span>
                    </label>
                    <label
                      className="MuiFormControlLabel-root"
                    >
                      <span
                        aria-disabled={false}
                        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-218 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                        onBlur={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={null}
                      >
                        <span
                          className="MuiIconButton-label"
                        >
                          <input
                            className="PrivateSwitchBase-input-221"
                            data-indeterminate={false}
                            onChange={[Function]}
                            type="checkbox"
                            value="sudeste"
                          />
                          <svg
                            aria-hidden="true"
                            className="MuiSvgIcon-root"
                            focusable="false"
                            role="presentation"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                      >
                        Sudeste
                      </span>
                    </label>
                    <label
                      className="MuiFormControlLabel-root"
                    >
                      <span
                        aria-disabled={false}
                        className="MuiButtonBase-root MuiIconButton-root PrivateSwitchBase-root-218 MuiCheckbox-root MuiCheckbox-colorSecondary MuiIconButton-colorSecondary"
                        onBlur={[Function]}
                        onDragLeave={[Function]}
                        onFocus={[Function]}
                        onKeyDown={[Function]}
                        onKeyUp={[Function]}
                        onMouseDown={[Function]}
                        onMouseLeave={[Function]}
                        onMouseUp={[Function]}
                        onTouchEnd={[Function]}
                        onTouchMove={[Function]}
                        onTouchStart={[Function]}
                        tabIndex={null}
                      >
                        <span
                          className="MuiIconButton-label"
                        >
                          <input
                            className="PrivateSwitchBase-input-221"
                            data-indeterminate={false}
                            onChange={[Function]}
                            type="checkbox"
                            value="sul"
                          />
                          <svg
                            aria-hidden="true"
                            className="MuiSvgIcon-root"
                            focusable="false"
                            role="presentation"
                            viewBox="0 0 24 24"
                          >
                            <path
                              d="M19 5v14H5V5h14m0-2H5c-1.1 0-2 .9-2 2v14c0 1.1.9 2 2 2h14c1.1 0 2-.9 2-2V5c0-1.1-.9-2-2-2z"
                            />
                          </svg>
                        </span>
                      </span>
                      <span
                        className="MuiTypography-root MuiFormControlLabel-label MuiTypography-body1"
                      >
                        Sul
                      </span>
                    </label>
                  </div>
                </article>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        className="MuiGrid-root MuiGrid-item MuiGrid-grid-xs-12 MuiGrid-grid-sm-12 MuiGrid-grid-md-9"
      >
        <div
          className="MuiGrid-root makeStyles-results-232"
        >
          <div
            className="MuiGrid-root MuiGrid-container MuiGrid-spacing-xs-3"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
